.github/workflows/update_references.yaml

```
@@ -0,0 +1,56 @@
+name: Update references
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
```

```suggestion
jobs:
  strategy:
    max_parallel: 1
```

To avoid race conditions with pulling and committing the screenshots causing the job to fail